examples/pizza-order/app/core/config.py

```
@@ -0,0 +1,32 @@
+
+class Settings(BaseSettings):
+    """Application settings"""
+
+    # Project
+    PROJECT_NAME: str = "pizza-order"
+    VERSION: str = "1.0.0"
+    ENVIRONMENT: str = "development"
+    DEBUG: bool = True
+
+    # Security
+    SECRET_KEY: str = "rGiDGpcJsGaCypQsGb1Yph4Vo_6ehlBsdwY7-5B5wUo"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)